setup.py

```diff
@@ -0,0 +1,12 @@
+from distutils.core import setup
+
+setup(
+    name='projectTemplate',
+    version='2021.12.14',
+    packages=['unittest'],
+    url='https://github.com/pyPadaiyal/projectTemplate',
+    license='Apache License 2.0',
+    author='padaiyal',
+    author_email='',
```

Comment:
The email field is left empty intentionally. Will fill it in later when we have an organizational e-mail address.